### ISSUE TITLE
Please support spyder 3.x

### ISSUE THREAD

**Issue by picca** [PierreRaybaut/guiqwt#66]
Hello,

spyder 3.x renamed the spyderlib module into spyder.

Could you support this inorder to have guiqwt/guidata and spyder3 available into Debian/Ubuntu.

Thanks

**Comment by stonebig** [comment]
same experiment replacing "spyderlib" per "spyder":
````
            import spyderlib.widgets.objecteditor  # analysis:ignore
    from spyderlib.widgets.internalshell import InternalShell
        # Eventually add an internal console (requires 'spyderlib')
                print('sift: spyderlib is outdated', file=sys.stderr)
    from spyderlib.widgets.objecteditor import oedit
    from spyderlib.widgets.objecteditor import oedit
    """Edit item data (requires `spyderlib`)"""
                from spyderlib.widgets.variableexplorer \
                from spyderlib.widgets import dicteditor
            from spyderlib.widgets.variableexplorer import arrayeditor
            from spyderlib.widgets import arrayeditor
from spyderlib.widgets.sourcecode.codeeditor import CodeEditor
# (see spyderlib/__init__.py for details)
guidata.py3compat (exact copy of spyderlib.py3compat)
# (see spyderlib/__init__.py for details)
spyderlib.qt.compat
# (copied from Spyder source code [spyderlib.qt])
# (see spyderlib/__init__.py for details)
# (copied from Spyder source code [spyderlib.qt])
# (see spyderlib/__init__.py for details)
# (see spyderlib/__init__.py for details)
# (see spyderlib/__init__.py for details)
        if 'spyderlib.spyder' in sys.modules:
            from spyderlib import __version__ as spyder_ver
````

would you mind proposing the patch picca ?

**Comment by PierreRaybaut** [comment]
Fixed in commit 2c178fe.
